apps/circuit-compiler/src/app/services/circomPluginClient.ts

```diff
@@ -313,15 +313,15 @@ export class CircomPluginClient extends PluginClient {
         if (path.indexOf('https://') === 0) {
           // Regular expression to match include statements and make deps imports uniform
           const includeRegex = /include "(.+?)";/g
-          const replacement = 'include "circomlib/circuits/$1";'
+          const replacement = 'include "/circomlib/circuits/$1";'
 
           dependencyContent = dependencyContent.replace(includeRegex, replacement)
         } else {
           if (!include.startsWith('circomlib') && !pathModule.isAbsolute(filePath) && !pathModule.isAbsolute(path)) {
           // if include is not absolute, resolve it using the parent path of the current file opened in editor
             const absIncludePath = pathModule.resolve('/' + filePath.slice(0, filePath.lastIndexOf('/')), '/' + path)
```

Comment:
`absIncludePath ` already has a `/` prepended here why do you know it here too: `/${absIncludePath}`